### ISSUE TITLE
Windows & Android client/Docker server sync returning Unexpected token < in JSON at position 0

### ISSUE THREAD

**Issue by BackedUpBooty** [laurent22/joplin#6060]
I've read through Issue #5287 but couldn't find anything I could see to implement a fix, completely possible I didn't understand it all, so if the fix is there, please accept my apologies and point me to it. 

I have joplin server set up in Docker which has been working without a hitch for at least 6 months, possibly longer. I have clients on Win11 (since December) Win10 and Android which have been syncing perfectly. 

Today none of the clients will sync with the server, though there have been no new updates. I can access my `joplin.domain.com` reverse proxy via browser fine, and log in to my admin and user accounts fine. It's just the apps which won't sync, and the windows clients both return 
```
Error. Please check that URL, username, password, etc. are correct and that the sync target is accessible. The reported error was:
Unexpected token < in JSON at position 0
```
with the Android app returning a slightly different
```
Error. Please check that URL, username, password, etc. are correct and that the sync target is accessible. The reported error was:
JSON Parse error: Unrecognized token '<' 
```

## Environment

Joplin server is 2.7.3
Windows Clients (both) are 2.6.1
Android client is 2.6.8

## Steps to reproduce

1. Start server
2. Hit 'synchronize' on any app
3. Error returned as above

## Describe what you expected to happen

The apps to connect to the server and sync.

## Logfile

Here's a pastebin of the windows app console outputs which just loop https://pastebin.com/5xiYeXfX

I can provide the full windows app logs if necessary. 

I don't have logfiles for the server in docker but can pull some (probably basic) ones from portainer - my own look through that doesn't seem to show any errors though.

**Comment by laurent22** [comment]
Probably the nginx proxy is returning an error page. You need to check what it is, either on the log or by making an api request.

**Comment by leematos** [comment]
@BackedUpBooty -- Laurent's theory sounds right to me. You aren't using S3 sync so #5287 is not relevant.

**Comment by BackedUpBooty** [comment]
Thanks for the replies, I appreciate you taking the time. 

> You aren't using S3 sync so #5287 is not relevant.

Yeah I figured but wanted to make sure I was reading what I could that may have been related to the issue.

> You need to check what it is, either on the log or by making an api request.

An initial look at the nginx error and access logs doesn't seem to have anything to do with joplin, there are some errors but a) those are specifically SWAG related and b) go back to before this issue started.  

Took me a minute to figure out how to do API requests, I think I found that it reported a Cloudflare Access sign-in issue (which is weird as I've had Access enabled for a long while) but disabling Access for this particular subdomain got my apps syncing again. 

Thanks for the pointers.
